chart/Chart.yaml

```diff
@@ -1,8 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: elemental/elemental-operator:%OPERATOR_VERSION%
+#!BuildTag: elemental/elemental-operator:%OPERATOR_VERSION%-build%RELEASE%
 apiVersion: v2
 name: elemental-operator
 description: Rancher Elemental Operator
-version: 0.0.0
-appVersion: 0.0.0
+version: "%OPERATOR_VERSION%"
+appVersion: "%OPERATOR_VERSION%"
```

Comment:
This assumes that the chart version is always in sync with the image version, and that is currently true but it migth not be?

Reply:
Also if this is not substituted by mistake (i.e. someone building the chart themselves), the original chart will still work with version 0.0.0 but this one wont as that value is not a valid value

Reply:
also again (lol sorry) this was definitevely distinct before as version would be a semver valid version, while appversion would include the version+hash for easy identification

Reply:
again, I do not have strong opinion here, I am fine having the version and appVersion set as  `0.0.0`, this will just make obs builds slightly more convoluted. We should apply the same criteria in both though. The replacement here IMHO is nice since this is something we can easily do in OBS so we use equivalent logic in both sides.